trunk/source/PEAtoBoogie/src/de/uni_freiburg/informatik/ultimate/pea2boogie/VerificationResultTransformer.java

```diff
@@ -134,6 +138,21 @@ public IResult convertTraceAbstractionResult(final IResult result) {
 			oldRes = (AbstractResultAtElement<?>) result;
 			reqCheck = (ReqCheck) ((IResultWithCheck) result).getCheckedSpecification();
 			isPositive = true;
+		} else if (result instanceof InvariantResult<?, ?>) {
+			invResult = (InvariantResult<?, ?>) result;
+			oldRes = (AbstractResultAtElement<?>) result;
+			final var check = invResult.getChecks().stream().filter(ReqCheck.class::isInstance).findFirst();
+			if (check.isEmpty()) {
+				// Not sure if such a case can ever occur or if to just return result here
+				return null;
```

Comment:
This will hide cases where -- due to settings -- the analysis produces invariants (might be costly). I think it would be better to return the result in this case.

Reply:
I'm returning the result now.